templates/firefox-ios.xliff

```diff
@@ -2536,11 +2597,11 @@
         <source>$(PRODUCT_NAME)</source>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>14.0</source>
+        <source>16.0</source>
       </trans-unit>
     </body>
   </file>
-  <file original="Client/Intro.strings" source-language="en" datatype="plaintext">
+  <file original="Shared/Supporting Files/Intro.strings" source-language="en" datatype="plaintext">
```

Comment:
If file name is changed, all existing localizations within that file will be lost